src/main/java/uk/ac/ebi/embl/gff3tools/validation/ValidationEngineBuilder.java

```diff
@@ -77,33 +62,23 @@ private Map<String, RuleSeverity> loadDefaultSeverities() {
             // load a properties file
             prop.load(input);
 
-            prop.forEach((k, v) -> {
-                String rule = (String) k;
-                RuleSeverity severity = RuleSeverity.valueOf((String) v);
-                severities.put(rule, severity);
+            prop.forEach((key, value) -> {
+                String k = (String) key;
+                String v = (String) value;
+
+                if (k.startsWith("rule")) {
```

Comment:
Not too happy we need to add prefixes to differentiate them.
Also, its not possible by doing this to validate if the user has a typo in the validation name.
We should error if the validation name does not exist.

I think ideally we should pre-load the validations, then check the names against what was loaded
so we can provide good user feedback.
This is why we had the loading and enabling in two different stages before.